src/build-data/oids.txt

```diff
@@ -379,6 +379,8 @@
 2.5.29.36 = X509v3.PolicyConstraints
 2.5.29.37 = X509v3.ExtendedKeyUsage
 1.3.6.1.5.5.7.1.1 = PKIX.AuthorityInformationAccess
+1.3.6.1.5.5.7.1.7 = PKIX.ipAddrBlocks
+1.3.6.1.5.5.7.1.8 = PKIX.autonomousSysIds
```

Comment:
Convention within the codebase (not adhered to 100%, admittedly) is for the part after the dot to be capitialized, eg `PKIX.IpAddressBlocks` / `PKIX.AutonomousSysIds`

Also here please add a comment referencing the RFC to assist future readers